modal/image.py

```diff
@@ -501,26 +533,28 @@ def pip_install_private_repos(
             )
 
         secret_names = ",".join([s.app_name if hasattr(s, "app_name") else str(s) for s in secrets])  # type: ignore
-        dockerfile_commands = ["FROM base"]
-        if any(r.startswith("github") for r in repositories):
-            dockerfile_commands.append(
-                f"RUN bash -c \"[[ -v GITHUB_TOKEN ]] || (echo 'GITHUB_TOKEN env var not set by provided modal.Secret(s): {secret_names}' && exit 1)\"",
```

Comment:
this is existing code so not related to this PR, but just noticed it and... it's pretty ugly?